space/core/components/editor/toolbar.tsx

```
@@ -3,13 +3,14 @@
 import React, { useEffect, useState, useCallback } from "react";
 // editor
 import { EditorRefApi, TEditorCommands, TNonColorEditorCommands } from "@plane/editor";
+import { useTranslation } from "@plane/i18n";
 // ui
+import { useTranslation } from "@plane/i18n";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)